splinterd/src/main.rs

```diff
@@ -485,25 +467,31 @@ fn main() {
 
     let matches = app.get_matches();
 
-    let log_level = match matches.occurrences_of("verbose") {
+    let clap_log_level = match matches.occurrences_of("verbose") {
```

Comment:
This change seems unnecessary

Reply:
fixed, left from when this was a larger more comprehensive change